### PR TITLE
release prep

### PR DESCRIPTION
**Pull request by roverflow**
Release prep pr

### CODE REVIEW

File: meta/runtime.yml [hunk 1]
```diff
@@ -46,4 +46,4 @@ plugin_routing:
       redirect: splunk.es.splunk_adaptive_response_notable_event
     adaptive_response_notable_events:
       redirect: splunk.es.splunk_adaptive_response_notable_events
-requires_ansible: ">=2.9.10"
```
Comment (KB-perByte) [inline meta/runtime.yml]: why are we changing this this needs a major release

File: changelogs/changelog.yaml [hunk 2]
```diff
@@ -67,3 +64,7 @@ releases:
         name: splunk_data_inputs_network
         namespace: ansible_collections.splunk.es.plugins.modules
     release_date: "2022-09-07"
+  2.2.0:
```
Comment (KB-perByte) [inline changelogs/changelog.yaml]: We are bumping up 2.2.0 over 2.0.0?  Where is 2.1.0?
Moreover, this should be a bugfix release.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -5,12 +5,34 @@ Splunk Enterprise Security Collection Release Notes
 .. contents:: Topics
 
 
+v2.2.0
+======
+
+Release Summary
+---------------
+
+Releasing 2.2.0 with some maintainance updates.
```
Comment (KB-perByte) [inline CHANGELOG.rst]: We can be a bit more assertive here

**Review by KB-perByte — CHANGES_REQUESTED** [review]

File: meta/runtime.yml (hunk 1 above)
Comment (roverflow) [reply meta/runtime.yml]: Will change this back

File: changelogs/changelog.yaml (hunk 2 above)
Comment (roverflow) [reply changelogs/changelog.yaml]: Will change this

**Review by KB-perByte — APPROVED** [review]

**Pull request merged**